syntax = "proto3";

option go_package = '../proto';

service ResumeCounterService{
  // 统计简历
  rpc CountResume(CountResumeRequest) returns(CountResumeResponse);
}


message CountResumeRequest{
  int64 user_id = 1;
  string name = 2;
  int32 type = 3;
  int32 status = 4;
}

message CountResumeResponse{
  int64 count = 1;
}